tests/fuzz/README.md

```diff
@@ -0,0 +1,35 @@
+What is it ?
+------
+
+This directory contains fuzz targets.
+Fuzz targets are simple codes using the library.
+They are used with a so-called fuzz driver, which will generate inputs, try to process them with the fuzz target, and alert in case of an unwanted behavior (such as a buffer overflow for instance).
+
+These targets were meant to be used with oss-fuzz but can be used in other contexts.
+
+This code was contributed by Philippe Antoine ( Catena cyber ).
+
+How to run ?
```

Comment:
Really minor, but putting a space before `?` or `:` or `;` is a very French thing and shocks the other 99% of the world.

Reply:
I did not know it was French only :-)

Fixed